models/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, archived, on_spotify: false)
+    super(publish_date, archived: archived)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    @on_spotify && Date.today.year - Date.parse(@publish_date).year > 10
+  end
```

Comment:
## This is a private method according to the [UML Diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png)

The method should check the `Item` class `can_be_archived?` and `@on_spotify` from `MusicAlbum`.
The better implementation would be : 

```
 private

 def can_be_archived?
    super && @on_spotify
  end
```